explore-analyze/machine-learning/nlp/ml-nlp-ner-example.md

```diff
@@ -117,7 +117,7 @@ Using the example text "Elastic is headquartered in Mountain View, California.",
 
 You can perform bulk {{infer}} on documents as they are ingested by using an [{{infer}} processor](elasticsearch://reference/enrich-processor/inference-processor.md) in your ingest pipeline. The novel *Les Misérables* by Victor Hugo is used as an example for {{infer}} in the following example. [Download](https://github.com/elastic/stack-docs/blob/8.5/docs/en/stack/ml/nlp/data/les-miserables-nd.json) the novel text split by paragraph as a JSON file, then upload it by using the [Data Visualizer](../../../manage-data/ingest/upload-data-files.md). Give the new index the name `les-miserables` when uploading the file.
 
-Now create an ingest pipeline either in the [Stack management UI](ml-nlp-inference.md#ml-nlp-inference-processor) or by using the API:
+Now create an ingest pipeline either from the [Ingest Pipelines](ml-nlp-inference.md#ml-nlp-inference-processor) page in {{kib}} or by using the API:
```

Comment:
Optional suggestion to keep it consistent with [this](https://github.com/elastic/docs-content/pull/3319/files#diff-617ce1bfe5bcb6b20b56e93eeb9b8c5a8f8704faeaecbd58bd294ceda65fd630R22).

```suggestion
Now create an ingest pipeline either from the [Ingest Pipelines](ml-nlp-inference.md#ml-nlp-inference-processor) management page in {{kib}} or by using the API:
```